### PR TITLE
Bank teller messaging updates and fixed small timing issue

### PR DESCRIPTION
**Pull request by wmahmoud**
Bank teller messaging updates and fixed small timing issue for tgames.lic

### CODE REVIEW

File: scripts/tgames.lic [hunk 1]
```diff
@@ -333,7 +335,7 @@ def lootcheck(tier, game_location)
 			if result =~ /It is worth (.*) seashells./
 				found_seashells = $1.to_i
 				fput "redeem my seashell"
-				sleep 0.1
+				pause 0.25
```
Comment (ondreian) [inline scripts/tgames.lic:338]: I'm curious why this change was necessary?  Wouldn't it be better to verify the result versus hoping the network roundtrip times match up?
```ruby
fput "redeem my seashell"
redeem_timeout = Time.now + 5 # seconds
wait_while { Char.right.noun.eql?("seashell") && Time.now < redeem_timeout }
fail "BOOM: tell Tysong something is wrong with redeeming" if Time.now > redeem_timeout
```

Comment (wmahmoud) [reply scripts/tgames.lic:338]: About 1:25 times, the original 0.1 second pause would not be enough for the game-obj to update, verified that much with a couple people. It would probably be better to just look at the hands, and I could update it to be that instead.

Comment (ondreian) [reply scripts/tgames.lic:338]: I think that would probably be best, at least if we are going to change the sleep, there is no guarantee even `0.25` is the correct duration to use, so it will invariably break for someone else again.

Comment (ondreian) [reply scripts/tgames.lic:338]: maybe @mrhoribu wants to put some thoughts here, this is originally his script.

**Comment by ondreian** [comment]
It it intended that this pr also has changes for bigshot.lic?

**Comment by wmahmoud** [comment]
That was my bad, just the tgames change was intended

File: scripts/tgames.lic [hunk 2]
```diff
@@ -332,8 +334,7 @@ def lootcheck(tier, game_location)
 			)
 			if result =~ /It is worth (.*) seashells./
 				found_seashells = $1.to_i
-				fput "redeem my seashell"
-				sleep 0.1
+				dothistimeout "redeem my seashell", 3, /^You redeem your #{found_seashells} Ebon Gate seashells/
```
Comment (mrhoribu) [inline scripts/tgames.lic:337]: This looks better and more feasible than previous code and gives more time for completion. Thanks.

**Review by mrhoribu — APPROVED** [review]
